[mingw64] [libplacebo] fail to build: Multiple definitions of various symbols with libspirv_cross_dll

build failure straight out of the box of libplacebo

``` bash
20:29:12 ┌ vulkan-loader git  ........................ [Recently updated]
20:29:14 ├ Installing uasm... done
0006-loader-cross-compile-static-linking-hacks.patch
        Patch could not be applied with `git am`. Continuing without patching.
20:29:22 ├ Running dependencies...
20:29:41 ├ Installing Vulkan-Headers...
20:29:41 ├ Running cmake...
20:29:45 ├ Running build...
20:29:45 ├ Running install...
20:29:46 ├ Building Vulkan-Loader...
20:29:46 ├ Running cmake...
20:29:49 ├ Running build...
20:30:02 ├ Running install...
20:30:02 ├ Stripping...
20:30:02 └ vulkan-loader-git  ................................. [Updated]
20:30:03   Running git clone for SPIRV-Cross ...
20:30:15   Running git update for SPIRV-Cross ...
20:30:16 ┌ SPIRV-Cross git  .......................... [Recently updated]
20:30:17 ├ Running meson...
20:30:23 ├ Running build...
20:31:10 ├ Running install...
20:31:10 ├ Stripping...
20:31:11 └ SPIRV-Cross-git  ................................... [Updated]
20:31:11   Running git clone for glslang ...
20:31:24   Running git update for glslang ...
20:31:25 ┌ glslang git  .............................. [Recently updated]
20:31:25 ├ Running dependencies...
20:31:50 ├ Running cmake...
20:31:55 ├ Running build...
20:38:18 ├ Running install...
20:38:19 ├ Stripping...
20:38:19 └ glslang-git  ....................................... [Updated]
20:38:19   Running git clone for shaderc ...
20:38:24   Running git update for shaderc ...
20:38:25 ┌ shaderc git  .............................. [Recently updated]
20:38:26 ├ Running dependencies...
20:38:38 ├ Running cmake...
20:38:48 ├ Running build...
20:44:50 ├ Running install...
20:44:51 ├ Stripping...
20:44:51 └ shaderc-git  ....................................... [Updated]
20:44:51   Running git clone for libplacebo ...
20:44:59   Running git update for libplacebo ...
20:45:00 ┌ libplacebo git  ........................... [Recently updated]
20:45:22 ├ Installing python-mako... done
20:45:25 ├ Installing python-setuptools... done
20:45:31 ├ Running meson...
20:45:58 ├ Running build...
Likely error (tail of the failed operation logfile):
C:/git/mab-full/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/git/mab-full/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../lib\libstdc++.a(string-inst.o):(.text$_ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE7reserveEy[_ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE7reserveEy]+0x0): multiple definition of `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::reserve(unsigned long long)'; C:/git/mab-full/local64/lib/libspirv-cross.dll.a(libspirv_cross_dll_d002312.o):(.text+0x0): first defined here
C:/git/mab-full/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/git/mab-full/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../lib\libstdc++.a(string-inst.o):(.text$_ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE9push_backEc[_ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE9push_backEc]+0x0): multiple definition of `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::push_back(char)'; C:/git/mab-full/local64/lib/libspirv-cross.dll.a(libspirv_cross_dll_d002317.o):(.text+0x0): first defined here
C:/git/mab-full/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/git/mab-full/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../lib\libstdc++.a(string-inst.o):(.text$_ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE14_M_replace_auxEyyyc[_ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE14_M_replace_auxEyyyc]+0x0): multiple definition of `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_replace_aux(unsigned long long, unsigned long long, unsigned long long, char)'; C:/git/mab-full/local64/lib/libspirv-cross.dll.a(libspirv_cross_dll_d002308.o):(.text+0x0): first defined here
C:/git/mab-full/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/git/mab-full/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../lib\libstdc++.a(string-inst.o):(.text$_ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE10_M_replaceEyyPKcy[_ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE10_M_replaceEyyPKcy]+0x0): multiple definition of `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_replace(unsigned long long, unsigned long long, char const*, unsigned long long)'; C:/git/mab-full/local64/lib/libspirv-cross.dll.a(libspirv_cross_dll_d002306.o):(.text+0x0): first defined here
C:/git/mab-full/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/git/mab-full/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../lib\libstdc++.a(string-inst.o):(.text$_ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE9_M_appendEPKcy[_ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE9_M_appendEPKcy]+0x0): multiple definition of `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_append(char const*, unsigned long long)'; C:/git/mab-full/local64/lib/libspirv-cross.dll.a(libspirv_cross_dll_d002314.o):(.text+0x0): first defined here
C:/git/mab-full/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/git/mab-full/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../lib\libstdc++.a(string-inst.o):(.text$_ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE6appendEPKc[_ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE6appendEPKc]+0x0): multiple definition of `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::append(char const*)'; C:/git/mab-full/local64/lib/libspirv-cross.dll.a(libspirv_cross_dll_d002310.o):(.text+0x0): first defined here
C:/git/mab-full/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/git/mab-full/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../lib\libstdc++.a(string-inst.o):(.text$_ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE6substrEyy[_ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE6substrEyy]+0x0): multiple definition of `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::substr(unsigned long long, unsigned long long) const'; C:/git/mab-full/local64/lib/libspirv-cross.dll.a(libspirv_cross_dll_d002252.o):(.text+0x0): first defined here
C:/git/mab-full/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/15.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/git/mab-full/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/15.2.0/libgcc_eh.a(unwind-seh.o):(.text+0x420): multiple definition of `_Unwind_Resume'; C:/git/mab-full/local64/lib/libspirv-cross.dll.a(libspirv_cross_dll_d000015.o):(.text+0x0): first defined here
collect2.exe: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
build failed. Check C:/git/mab-full/build/libplacebo-git/build-64bit/ab-suite.build.log
This is required for other packages, so this script will exit.
20:47:21   Creating diagnostics file...

```

[logs.zip](https://github.com/user-attachments/files/25586954/logs.zip)

--- a/build/media-suite_helper.sh
+++ b/build/media-suite_helper.sh
@@ -1391,11 +1391,9 @@ do_meson() {
     extra_script pre meson
     [[ -f "$(get_first_subdir -f)/do_not_reconfigure" ]] &&
         return
-    local default_libs
-    [[ $ffmpeg =~ "both" ]] && default_libs="both" || default_libs="static"
     # shellcheck disable=SC2086
     PKG_CONFIG="pkgconf --static --keep-system-libs --keep-system-cflags" CC=${CC/ccache /}.bat CXX=${CXX/ccache /}.bat \
-        log "meson" meson setup "$root" --default-library="$default_libs" --default-both-libraries=static --buildtype=release \
+        log "meson" meson setup "$root" --default-library=static --default-both-libraries=static --buildtype=release \
         --prefix="$LOCALDESTDIR" --backend=ninja $bindir "$@" "${meson_extras[@]}"
     extra_script post meson
     unset meson_extras

